src/components/toggle/toggle.svelte

```diff
@@ -106,9 +93,9 @@
   }
 
   .leo-toggle {
-    --width: var(--leo-toggle-width, 56px);
+    --width: var(--leo-toggle-width, 52px);
     --height: var(--leo-toggle-height, 32px);
-    --padding: var(--leo-toggle-padding, 2px);
+    --padding: var(--leo-toggle-padding, 4px);
```

Comment:
Should this use the `--leo-spacing-s` instead of `4px`?

Reply:
Hmm, probably, yeah!